documentation/src/docs/deployment/configuration-and-provisioning.md

```diff
@@ -0,0 +1,380 @@
+# Pravega Configuration and Provisioning Guide
+
+This document summarizes the most important aspects to consider for configuring and provisioning a
+production Pravega Cluster in an infrastructure-agnostic way. Note that this document does not cover all the 
+parameters in the configuration of Pravega as they are [already documented](/config.properties). Similarly,
+this is not a guide to [deploy Pravega on Kubernetes](https://github.com/pravega/pravega-operator).
```

Comment:
Rephrase: "For Kubernetes users, instead see our guide to deploy Pravega on Kubernetes"

Reply:
I have split the content into few smaller files, so this piece of text has been removed.